instrumentation_sql.go

```diff
@@ -227,11 +227,38 @@ func (conn *wrappedSQLConn) ExecContext(ctx context.Context, query string, args
 }
 
 func (conn *wrappedSQLConn) CheckNamedValue(d *driver.NamedValue) error {
+	defer func() {
```

Comment:
If any unexpected error occurs, we return error = nil, so we don't break customers code.